SamplifyAPIClient/Roles.cs

```diff
@@ -0,0 +1,30 @@
+using System.Runtime.Serialization;
+
+namespace Dynata.SamplifyAPIClient
+{
+    [DataContract]
+    public class Roles
+    {
+        [DataMember(Name = "id")]
+        public string ID { get; set; }
```

Comment:
public string ID --- Should this be a string or an int?

Reply:
This should be a string. The example of ID here is `PROJECT_MANAGER`